Source/BugsnagCrashReport.m

```diff
@@ -64,18 +64,24 @@
 }
 
 NSString *BSGParseErrorClass(NSDictionary *error, NSString *errorType) {
```

Comment:
We could annotate this return value as non-null to change having null returned to a build error.